.travis.yml

```
@@ -0,0 +1,3 @@
+language: python
+install: pip install flake8
+script: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

```suggestion
script: flake8 tools/ --count --select=E9,F63,F7,F82 --show-source --statistics
```

The rest of the codebase is in C/C++, no point looking there.

---

I bet there is no performance difference but there is a safety difference but implementing the suggestion anyway.